### PR TITLE
Changed edit and cancel button background colors

### PR DESCRIPTION
**Pull request by KyleD0711**

### CODE REVIEW

File: src/components/admin/maintain/semesters/SemesterDialogBody.vue
Comment (dbnorth) [inline src/components/admin/maintain/semesters/SemesterDialogBody.vue:1]: Went is Add it does not right justify the buttons

Comment (KyleD0711) [reply src/components/admin/maintain/semesters/SemesterDialogBody.vue:1]: Fixed

**Review by dbnorth — CHANGES_REQUESTED** [review]
Need to fix the justification on add

**Review by dbnorth — APPROVED** [review]
Looks good

**Pull request merged**